packages/node/src/jsonLayout.ts

```diff
@@ -1,14 +1,18 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 
+import correlationId from "correlation-id";
 import stringify from "json-stringify-safe";
 import log4js, { LoggingEvent } from "log4js";
 import util from "util";
 
 function jsonFormat(logEvent: LoggingEvent) {
+    // The correlation id is undefined when no set in the current call stack
```

Comment:
Doesn't it generate a new one if it wasn't set explicitly?

Reply:
I think it does and this is not good. We would log an ID that is very misleading.

Reply:
Nop at this place it only reads the ID no generation here.